x-pack/plugins/apm/server/lib/apm_telemetry/apm_telemetry.ts

```diff
@@ -16,6 +16,7 @@ export enum AgentName {
   Java = 'java',
   NodeJs = 'nodejs',
   JsBase = 'js-base',
+  RumJs = 'rum-js',
   Ruby = 'ruby',
   GoLang = 'go'
 }
```

Comment:
Is it possible to merge this `AgentName ` enum and the `AgentName` type in APMDoc?